code/game/objects/items/weapons/lighters.dm

```diff
@@ -6,7 +6,9 @@
 	lefthand_file = 'icons/mob/inhands/lighter_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/lighter_righthand.dmi'
 	icon_state = "lighter-g"
+	base_icon_state = "lighter"
 	item_state = "lighter-g"
+	var/base_item_state = "lighter"
```

Comment:
This is never different from the `base_icon_state`, just change all `base_item_state` into `base_icon_state`